### PR TITLE
Fix for when location and live boxes have no data

### PR DESCRIPTION
**Pull request by danleedham**
Well let's get the ball rolling on this one for #82 

I have two options: 
One's programmatic (see branch: fix-empty-locaiton) which sets the display to false if nothing's entered
This one is my preferred option which just sets the element not to display if it's empty.

### CODE REVIEW

File: public/css/theme.css [hunk 1]
```diff
@@ -209,6 +209,10 @@ h1, h2, h3, h4 {
     background-image: linear-gradient(to bottom, rgba(255, 255, 255, 0.3), rgba(255, 255, 255, 0) 50%, rgba(0, 0, 0, 0.12) 51%, rgba(0, 0, 0, 0.04));
 }
 
+.bug h1:empty {
```
Comment (stephenrob) [inline public/css/theme.css:212]: Is this support within the CEF version on CasparCG? No worries if its only in the latest beta as that will be fine for us to run over roses

**Review by stephenrob — COMMENTED** [review]
Compatibility with CEF version running in CasparCG? Which version did you test against @danleedham ?

This is a very clean approach to it though and would be better than a programmatic one.

File: public/css/theme.css (hunk 1 above)
Comment (danleedham) [reply public/css/theme.css:212]: 2.0.7 (Latest Stable?)

no text

with text

**Review by stephenrob — APPROVED** [review]
Happy to merge in with those screenshots. My memory of CSS and what works with CasparCG is a bit rusty.

**Pull request merged**